rke/templates/calico.go

```diff
@@ -675,7 +675,7 @@ data:
   calico_backend: "bird"
 
   # Configure the MTU to use
-{{- if ne .MTU ""}}
+{{- if ne .MTU 0}}
   veth_mtu: "{{.MTU}}"
 {{- else }}
   veth_mtu: "1440"
```

Comment:
why are we defaulting to 1440?

Reply:
Thats the default of calico install template:

```
curl -s https://docs.projectcalico.org/v3.8/manifests/calico.yaml | grep mtu
```